### ISSUE TITLE
$ref are not resolved with get method

### ISSUE THREAD

**Issue by elcamlost** [jhthorsen/json-validator#262]
* JSON::Validator version: 5.0
* Perl version: 5.26
* Operating system: CentOS 8

### Steps to reproduce the behavior

```perl
use 5.026;
use JSON::Validator;
use Data::Dumper;

my $jv = JSON::Validator->new;
$jv->schema({x => {'$ref' => '#/y'}, y => {'type' => 'string'}});

say Dumper($jv->schema->get('/x'));
say Dumper($jv->get('/x'));
```

### Expected behavior

$jv->get('/x') prints

```
$VAR1 = {
          'type' => 'string'
        };
```

### Actual behavior

$jv->get('/x') prints

```
$VAR1 = {
          '$ref' => '#/y'
        };
```

**Comment by elcamlost** [comment]
My real goal is to get resolved specification with $c->openapi->spec

**Comment by jhthorsen** [comment]
The title is not correct, but I can see that the current behavior is confusing so I will change it to do what you expect.

This is the behavior right now, but will be changed in 5.01:
$jv->get('/x') returns {"$ref":"..."}
$jv->get('/x/type') returns "string"

**Comment by jhthorsen** [comment]
This is now changed/fixed in d367eb8e3c22a27e4c5bfaea21faf12db8349e88.
